modules/mvt/src/lib/binary-vector-tile/vector-tile-layer.d.ts

```diff
@@ -5,7 +5,11 @@ import VectorTileFeature from './vector-tile-feature';
  * a vector tile layer. It has the same semantics
  * as the original library (https://github.com/mapbox/vector-tile-js)
  * except that the features it contains expose their data
- * in a binary optimized format, rather than geoJSON
+ * in a binary optimized format, rather than GeoJSON
+ *
+ * The VectorTileLayer class is unmodified from the original source,
+ * except that the `feature` method passes through an additional
+ * `firstPassData` parameter to the VectorTileFeature
```

Comment:
@kylebarron as requested, I've marked which code is different from the original Mapbox implemention